### ISSUE TITLE
请问有人碰到过吞字吞音的情况吗？

### ISSUE THREAD

**Issue by MisakaMikoto96** [PlayVoice/vits_chinese#148]
比如一个汉字的声母被吞，或者韵母尾部听起来有丢失，这种情况是因为什么呢？
对比了它和gt的duration，发现duration是一致的，但就是高低频里对声母的发音或者韵母的发音不好。

请问有人碰到过这种情况吗？

**Comment by xinge333** [comment]
对齐没问题吗

**Comment by MisakaMikoto96** [comment]
> 对齐没问题吗

是的 对比 看了下生成的音频和ground truth的音频，字与字之间的duration基本上都一致，但频谱似乎还原得不太好，要么高频（声母），要么低频（韵母尾音）不行。

**Comment by MisakaMikoto96** [comment]
> 对齐没问题吗

是的 对比 看了下生成的音频和ground truth的音频，字与字之间的duration基本上都一致，但频谱似乎还原得不太好，要么高频（声母），要么低频（韵母尾音）不行。

**Comment by ScottishFold007** [comment]
这个情况很常见，是不是下面的情况：
1、模型训练不足：
如果模型还没学到位，可能就会出现发音不准确的情况。这时候，模型可能还没完全掌握如何处理不同的声母和韵母。
2、数据集问题：
如果你用来训练的数据集里，某些声母或韵母的例子不够多，或者录音质量不够好，模型就可能学不会怎么正确发音。
3、音频处理问题：
在预处理或者后处理阶段，如果对音频的处理不当，也可能导致发音丢失。比如，如果噪声去除做得太过头了，可能会把一些细微的发音也给去掉。
